app/translations/en.json

```diff
@@ -752,6 +752,8 @@
   "app.containers.Questions.thisIsNewCommunity":
     "This is a new community, you can be the first post a question. Your question may become the most popular.",
   "app.containers.Questions.readMyFeed": "Read my feed",
+  "app.containers.Questions.topQuestionPopover":
+    "A question that specified as top and frequently asked at this community.",
```

Comment:
Maybe let's change the wording to be simple and similar to Russian version - "Top question of this community"